types/src/front/api_handlers/public/data_sources.ts

```diff
@@ -41,7 +41,7 @@ export const PostDataSourceDocumentRequestBodySchema = t.intersection([
   }),
   t.partial({
     title: t.string,
-    mimeType: t.string,
+    mime_type: t.string,
```

Comment:
this is actually inconsistent with tables api - we should choose  mimeType or mimeType for all endpoints

Reply:
I take it you mean mimeType or mime_type ;)
IMO we go for snake case, as is the direction we'd like to take across the codebase

Reply:
let's go for mime_type in tables api